src/entities/reactions.rs

```diff
@@ -84,50 +84,42 @@ pub fn get_reactions(conn: &PgConnection, message_id: i64) -> HashMap<String, Ve
 
     let mut reactions_group = HashMap::<String, Vec<SavedReaction>>::new();
     for r in reactions.iter() {
-        if !reactions_group.contains_key(&r.reaction) {
-            reactions_group.insert(r.reaction.clone(), Vec::new());
-        }
-
         reactions_group
-            .get_mut(&r.reaction)
-            .expect("Couldn't get mutex")
```

Comment:
note: None is returned when the key is not present in the map, there is no mutex involved